packages/ui/src/DateTimeActionSheet.tsx

```diff
@@ -180,47 +181,60 @@ export function DateTimeActionSheet({
   onChange,
   visible,
   onDismiss,
+  transformValue,
 }: DateTimeActionSheetProps) {
   const {theme} = useContext(ThemeContext);
 
   // Accept ISO 8601, HH:mm, or hh:mm A formats. We may want only HH:mm or hh:mm A for mode=time
   let m;
   if (value) {
-    m = dayjs(value, ["YYYY", "YYYY-MM-DD", "HH:mm", "hh:mm A"]);
+    m = dayjs(value, [
+      "YYYY",
+      "YYYY-MM-DD",
+      "HH:mm",
+      "hh:mm A",
+      transformValue?.options?.transformFormat,
+    ]).tz(transformValue?.options?.timezone);
```

Comment:
if you pass in undefined here due to the optionals, does it default to local?

Reply:
I must have missed this question but yes. Undefined returns to the default local display